dcos_tunnel/cli.py

```diff
@@ -534,7 +542,7 @@ def run_vpn(command, output_file):
 
 
 def _vpn(port, config_file, user, privileged, ssh_port, host,
-         openvpn_container, vpn_client):
+         openvpn_container, vpn_client, docker_cmd='docker'):
```

Comment:
docopt already fills these out for you. Also, I think it's actually useful to receive a None here. See later comment.